### PR TITLE
Add onlcusive mapping of event_status [SDCP-749]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: server/cp/ingest/parser/cp_onclusive.py [hunk 1]
```diff
@@ -88,6 +88,19 @@ def parse(self, content, provider=None):
                 # remove duplicates
                 item["anpa_category"] = unique(category)
                 item["subject"] = unique(item["subject"])
+
+                # update event status SDCP-749
+                if item.get("is_provisional", False):
+                    eocstat_map = get_resource_service("vocabularies").find_one(
```
Comment (petrjasek) [inline server/cp/ingest/parser/cp_onclusive.py]: imo fetching it for every item is not very efficient, would be needed just once

Comment (petrjasek) [reply server/cp/ingest/parser/cp_onclusive.py]: and you can use `get_items` method which will only return active ones so you don't have to filter it further

Comment (devketanpro) [reply server/cp/ingest/parser/cp_onclusive.py]: all right !

**Comment by devketanpro** [comment]
fixed on :- https://github.com/superdesk/superdesk-planning/pull/1970

**Pull request closed without merging**